src/components/Lobby/GameView/PlayerView/PlayerView.tsx

```diff
@@ -87,6 +88,18 @@ export default function PlayerView(props: PlayerViewProps): JSX.Element {
 		SocketEventEmitter.vote(props.socket, vote);
 	}
 
+	const getVoteTargets = (): Player[] => {
+		let voteTargets = props.lobby.game.players;
```

Comment:
I don't understand why you init `voteTargets`  `withprops.lobby.game.players` and eplace it after `voteTargets = removeSelf(voteTargets);`